src/v0/destinations/mp/transform.js

```diff
@@ -70,35 +67,25 @@ const responseBuilderSimple = (payload, message, eventType, destConfig) => {
   response.method = defaultPostRequestConfig.requestMethod;
   response.userId = message.userId || message.anonymousId;
   response.body.JSON_ARRAY = { batch: JSON.stringify([removeUndefinedValues(payload)]) };
-  const { apiSecret, serviceAccountUserName, serviceAccountSecret, projectId, dataResidency } =
-    destConfig;
+  const { dataResidency } = destConfig;
   const duration = getTimeDifference(message.timestamp);
   switch (eventType) {
     case EventType.ALIAS:
     case EventType.TRACK:
     case EventType.SCREEN:
-    case EventType.PAGE:
-      if (
-        !apiSecret &&
-        !(serviceAccountUserName && serviceAccountSecret && projectId) &&
-        duration.days <= 5
-      ) {
-        response.endpoint =
-          dataResidency === 'eu' ? `${BASE_ENDPOINT_EU}/track/` : `${BASE_ENDPOINT}/track/`;
-        response.headers = {};
-      } else if (duration.years > 5) {
+    case EventType.PAGE: {
```

Comment:
Remove redundant block
```suggestion
    case EventType.PAGE: 
```

Reply:
'{}' is required because we are declaring variables within the block of switch case.

Reply:
shouldn't be required, can you re-check ? just check `merge` case

Reply:
In merge case, lint rule `no-case-declarations` was disabled. I have fixed if for merge case too.
https://eslint.org/docs/latest/rules/no-case-declarations